### PR TITLE
Mustafaansarii patch 2

### PR DESCRIPTION
**Pull request by mustafaansarii**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5100"></a>

### CODE REVIEW

File: data/mustafaansarii.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    { "icon": "FaInstagram", "url": "https://www.instagram.com/mustafa.nds/" }
+  ],
+  "testimonials": [
+    "Rahul kr",
+    "Sachin singh",
+    "nikhil raj",
+    "Ankit kr"
+    
+  ],
+  "avatar": "https://github.com/mustafa-nds.png",
```
Comment (swagataroy30) [inline data/mustafaansarii.json:29]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

**Review by SurajPratap10 — APPROVED** [review]

**Review by mahfuztbt — APPROVED** [review]
looks good

File: data/mustafaansarii.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/mustafa-ansari-75059a237/" },
+    { "icon": "FaInstagram", "url": "https://www.instagram.com/mustafa.nds/" }
+  ],
+  "testimonials": [
+    "Rahul kr",
+    "Sachin singh",
+    "nikhil raj",
+    "Ankit kr"
+    
+  ],
```
Comment (kumarsonsoff3) [inline data/mustafaansarii.json:28]: This is not how testimonials work, ask your testimonials to refer [these docs](https://linkfree.eddiehub.io/docs/how-to-guides/testimonials) and follow the required steps, then you can add their GitHub usernames here.

For now, it's better to remove this from your `json`. 

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**